### ISSUE TITLE
There is no sound when finding the slot...

### ISSUE THREAD

**Issue by ZhangchenZhao** [pcomputo/Whole-Foods-Delivery-Slot#30]
I am pretty sure that I turned on the sound and the program went well. But when it found the slot on the amazon website, there was no sound.

**Comment by pcomputo** [comment]
you can try going through the Readme, if that doesn't help there are many issues in this repo related to this that have been solved. It'll be helpful if you give them a quick shot. If nothing else works out, I am glad to take a look. Also, your computer specs will be really helpful to know whats going on.

**Comment by ZhangchenZhao** [comment]
Hi Pooja,

Thanks for the reply! Your software is extremely helpful and I already
shared it with my friends.

When I ran whole_foods_delivery_slot_chrome.py, there is no sound when it
finds the slot. Instead, it prints "SLOTS OPEN!!" and the following error
message "'say' is not recognized as an internal or external command,
operable program or batch file."

When I ran autobuy, it stopped at the last step with the following error
message (attached). I also attached my computer specs.

Thank you again for your help. Stay home and stay safe!

Best,
Zhangchen
